config/schema.yml

```diff
@@ -98,6 +98,7 @@ propel:
     id: { type: integer, required: true, primaryKey: true, foreignTable: object, foreignReference: id, onDelete: cascade, inheritanceKey: true }
     object_id: { type: integer, foreignTable: object, foreignReference: id, onDelete: cascade }
     usage_id: { type: integer, foreignTable: term, foreignReference: id, onDelete: setnull }
+    language: varchar(50)
```

Comment:
I realised that `language` is a reserved word in MySQL, so I'll change the column name to `lang` instead. I had some issues in MySQL Workbench because of this. This should avoid any issues in the future if this column needs to be modified. I'll also test and modify my other code to reflect this change.

```php
<?php

/*
 * This file is part of the Access to Memory (AtoM) software.
 *
 * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
 * it under the terms of the GNU Affero General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * Access to Memory (AtoM) is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
 */

/*
 * Add lang column (language) to digital_object table for use with subtitles.
 *
 * @package    AccesstoMemory
 * @subpackage migration
 */
class arMigration0190
{
  const
    VERSION = 190, // The new database version
    MIN_MILESTONE = 2; // The minimum milestone required

  public function up($configuration)
  {
    // Add extra column, lang
    QubitMigrate::addColumn(
        QubitDigitalObject::TABLE_NAME,
        'lang VARCHAR(50)',
        ['after' => 'usage_id']
    );
  
    return true;
  }
}

```

Just to clarify, do you want the results of `propel:build-sql` and `propel:build-model` included in the PR? These include modifications to these files:

`/usr/share/nginx/atom/lib/model/map/DigitalObjectTableMap.php`
`lib/model/om/BaseDigitalObject.php`
`data/sql/lib.model.schema.sql`

I originally excluded these since I wasn't sure if these tasks were run as part of initial setup.

As an aside, I'm also seeing modifications to `actor`, `object`, `term`, and `user` tables and classes when these tasks are run.

Reply:
Thanks for checking @mInnes-archives,

> Just to clarify, do you want the results of propel:build-sql and propel:build-model included in the PR?

Yes, please. These tasks are not part of the installation/upgrade process.

> As an aside, I'm also seeing modifications to actor, object, term, and user tables and classes when these tasks are run.

Good catch! Please, commit them too. Long story short, the builder has never been consistent ordering some stuff and it may reintroduce those changes the next time the models are built; but I'd like to check if it keeps happening and I think this is a good time.